mbf_costmap_nav/include/mbf_costmap_nav/footprint_helper.h

```diff
@@ -66,6 +66,17 @@ class FootprintHelper
                                              const std::vector<geometry_msgs::Point>& footprint_spec,
                                              const costmap_2d::Costmap2D&, bool fill);
 
+    /**
+   * @brief  Supercover algorithm is a modified Bresenham which prints ALL the points (not only one point per axis) the ideal line contains
+   * ref: http://eugen.dedu.free.fr/projects/bresenham/
```

Comment:
License is ok?

Reply:
I have sent an email to the author and his response:
```
Hi,

The code is free, use GPL if you can, or MIT etc.  I am glad it is
useful, and would be glad if you can credit me.

Best regards,
Eugen
```

Reply:
Hi, @RainerKuemmerle can you re-review? We would like to integrate this feature in mbf in a couple of days and your comments and approval  are very important for us.
Thanks in advance